cypress/README.md

```diff
@@ -9,12 +9,23 @@ We aim to keep the code reusable but also to leverage the cypress framework as m
 - Use the default cypress commands e.g. cy.get().
 - Code reusability is good but ease of understanding is better.
 - Use custom cypress commands, see `/support/commands.js`.
-- If you're using a let, var or const in any test/helper/command in this repo, you're probably doing it unecessarily, use cypress aliasing or chain requests. 
+- If you're using a let, var or const in any test/helper/command in this repo, you're probably doing it unecessarily, use cypress aliasing or chain requests.
 - If trying to reuse tests always use cypress commands. i.e. don't export and import const's within cypress. Config is a sensible exception to this rule.
 - Before writing a long/complex test, look to see if similar ones could be reused.
 - Once you've written a test consider where else it might be helpful. For example testing a 200 response is useful on all requests and visits, so why not add it there in an additional PR (not in the same one to keep our PRs small and therefore fast to merge).
 - Work together to consolidate the tests we have. Between the many tests in this folder are nearly all the tests we need, but are they as routinely used as they could be?
 - Write as many tests as you please, but don't make them atomic. i.e. use .and() instead of a second test.
--- The runtime of tests matters so make each as fast as you can, avoid repeating expensive things like visit()s and request()s as much as possible.
--- Although the runtime of each test matters and making each test not too small is good practice always tend towards full coverage rather than limiting the scope of testing.
+  -- The runtime of tests matters so make each as fast as you can, avoid repeating expensive things like visit()s and request()s as much as possible.
+  -- Although the runtime of each test matters and making each test not too small is good practice always tend towards full coverage rather than limiting the scope of testing.
 - We could write many more best practices but cypress already did: [Cypress.io Best Practices](https://docs.cypress.io/guides/references/best-practices.html)
+
+## How to run LIVE/TEST environment Cypress tests locally ?
+
+Run Cypress test for LIVE: `npm run test:e2e:live_env`
```

Comment:
This is unnecessarily verbose, add to the documentation that there are a range of script options available in the package.json. By duplicating the scripts here we will make the docs go stale and we discourage people from reading an important file (package.json).